torch/csrc/jit/python/script_init.cpp

```diff
@@ -2206,10 +2206,6 @@ void initJitScriptBindings(PyObject* module) {
       logging::LoggerBase,
       std::shared_ptr<logging::NoopLogger>>(m, "NoopLogger")
       .def(py::init<>());
```

Comment:
what's the difference between python/script_init and python/init?

Reply:
init: jit bindings
script_init: jit script bindings